sql_generators/active_users_aggregates_v3/templates/desktop_query.sql

```diff
@@ -2,16 +2,31 @@
 WITH todays_metrics AS (
   SELECT
     client_id,
-    activity_segments_v1 AS segment,
-    '{{ app_value }}' AS app_name,
+    CASE
+    WHEN BIT_COUNT(days_visited_1_uri_bits)
+      BETWEEN 1
+      AND 6
+      THEN 'infrequent_user'
+    WHEN BIT_COUNT(days_visited_1_uri_bits)
+      BETWEEN 7
+      AND 13
+      THEN 'casual_user'
+    WHEN BIT_COUNT(days_visited_1_uri_bits)
+      BETWEEN 14
+      AND 20
+      THEN 'regular_user'
+    WHEN BIT_COUNT(days_visited_1_uri_bits) >= 21
+      THEN 'core_user'
+    ELSE 'other'
+    END AS segment,
+    'Firefox Desktop' AS app_name,
```

Comment:
It seems like these segments are using "unqualified" usage (since they don't account for active hours), while the mobile segments have been replaced with `days_active_bits`. Is that the case? If so, what's the reason for the difference?

Reply:
Update: I think we should just use `days_active_bits` like we're using for mobile. I updated [the proposal](https://docs.google.com/document/d/1G_pJs62c8mGOxt1eGdzc_-pwG_ShE3hAb-N4Lmp9A8M/edit) to document this change.

Reply:
@bochocki To clarify, the segments would be defined by (`days_since_visited_1_uri` + `days_active_bits`), or only `days_active_bits`?

Reply:
The "active" definition is "total_uris > 0 and active_hours > 0" so I think `days_active_bits` alone should already account for `days_visited_1_uri_bits`. Does that seem right to you?

Reply:
Never mind, days_active_bits suffices